### DISCUSSION TITLE
level line style

### DISCUSSION THREAD

**Discussion by xuorel54** [freeplane/freeplane#1497] [Help]
Hello
I've set up level styles.
For each level I have set a line colour.
However, when I add nodes to my map, all sibling node lines have different colours, and all child nodes have the same colour.

I set automatic level styles with predefined edges color 
- blue for level 1
- orange for level 2
- green for level 3
- black for level 4

![image](https://github.com/freeplane/freeplane/assets/42151603/575b97ba-1947-457e-8bf6-4b144524be41)

however, when a add new nod, the edges all have the same colour ( randomly determined)

![image](https://github.com/freeplane/freeplane/assets/42151603/34377cbe-2a42-4cd9-a0f2-b9dc96741c59)

How can I achieve the desired result? 
Thanks

**Comment by i-plasm** [comment]
Try going to ` View -> Controls -> Tool panel `, and in the "Node Style" section set "Automatic edge color" to disabled.

**Reply by i-plasm** [reply]
(Edited to correct the menu for the Tool panel: View -> Controls -> Tool panel)

**Reply by xuorel54** [reply]
it doesn't fix my issue

**Comment by euu2021** [comment]
Please upload an example map, so we can show the step by step.

**Comment by i-plasm** [comment]
Try this: Go to ***Preferences -> Behaviour***. Uncheck these "preserving" format options

![pref-styles](https://github.com/freeplane/freeplane/assets/144830522/bc457cc9-ceef-435c-9690-9aecb15d99d9)
